src/Tables/organisms/TypeTable.tsx

```diff
@@ -156,7 +158,8 @@ const TypeTable: React.FC<IProps> = ({
                 columnConfig,
                 hasStatus,
                 hasThumbnail,
-                hasTypeIcon
+                hasTypeIcon,
+                closeText,
               }}
```

Comment:
Comma after closeText is not needed.

Also put another snapshot by changing CLOSE into Japanese.

Reply:
Done